library/extractor/src/main/java/com/google/android/exoplayer2/extractor/ogg/OggPageHeader.java

```diff
@@ -59,6 +61,10 @@
 
   private final ParsableByteArray scratch = new ParsableByteArray(MAX_SEGMENT_COUNT);
 
+  public OggPageHeader(long streamSerialNumber) {
+    expectedStreamSerialNumber = streamSerialNumber;
+  }
```

Comment:
Should we create both parameterised and default constructors here? The default constructor can call the parameterised one with default value for streamSerialNumber. The default value can be C.INDEX_UNSET or create a new constant if wanted.

Reply:
Javadoc is missing.